src/Wallabag/CoreBundle/Resources/views/themes/common/Static/howto.html.twig

```diff
@@ -147,6 +147,10 @@
                                 <td>{{ 'howto.shortcuts.toggle_favorite'|trans }}</td>
                             </tr>
                             <tr>
+                                <td><code>t</code></td>
+                                <td>{{ 'entry.view.left_menu.add_a_tag'|trans }}</td>
```

Comment:
This should better be `howto.shortcuts.add_a_tag`.
Which means a new line in each translated file.